examples/plotting/spans_and_strips.py

```diff
@@ -1,3 +1,12 @@
+''' An example demonstration the use of `hspan`, `wspan`, and `vstrip`.
+It also uses the `HoverTool`.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.HoverTool
```

Comment:
The hover tool is secondary for this example, I'd suggest refs for `bokeh.plotting.figure.hspan` and the other glyphs instead.